coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentControllerBase.java

```diff
@@ -394,11 +394,13 @@ protected ModelAndView proposalDevelopmentPageSave(ProposalDevelopmentDocumentFo
     protected ModelAndView narrativePageSave(ProposalDevelopmentDocumentForm form, boolean canEdit) throws Exception {
         ProposalDevelopmentDocument document = (ProposalDevelopmentDocument) getDocumentService().getByDocumentHeaderId(form.getDocument().getDocumentNumber());
         if (canEdit) {
-            if (new ProposalDevelopmentDocumentRule().processAttachmentRules(form.getProposalDevelopmentDocument())) {
+            if ((new ProposalDevelopmentDocumentRule().processAttachmentRules(form.getProposalDevelopmentDocument()))
+                    && (new ProposalDevelopmentDocumentRule().processPersonnelAttachmentDuplicates(form.getProposalDevelopmentDocument()))) {
             	form.getProposalDevelopmentAttachmentHelper().handleNarrativeUpdates(form, document);
             	form.getProposalDevelopmentAttachmentHelper().handleInstAttachmentUpdates(form, document);
             	form.getProposalDevelopmentAttachmentHelper().handlePersonBioUpdates(form, document);
                 document.getDevelopmentProposal().setProposalAbstracts(form.getDevelopmentProposal().getProposalAbstracts());
+                document.setDocumentNextvalues(form.getProposalDevelopmentDocument().getDocumentNextvalues());
```

Comment:
This is scary. First not related to this jira, secondly I'd also guess this means that this is an area of PD that can be modified in each locking section since I don't think attachments is the only thing that uses it. I think we need to do more here to merge the 2 lists when we do this.